distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -561,3 +568,24 @@ get_short_type_name typ = @Builtin_Method "Meta.get_short_type_name"
    - constructor: the constructor to get the declaring type of.
 get_constructor_declaring_type : Any -> Any
 get_constructor_declaring_type constructor = @Builtin_Method "Meta.get_constructor_declaring_type"
+
+instrumentor_builtin op args = @Builtin_Method "Meta.instrumentor_builtin"
+
+# Builder to create instrumentation for a function
```

Comment:
can we ensure these are all tagged `PRIVATE` and `ADVANCED` so they don't show up in the component browser.